js/yoast-acf-analysis.js

```diff
@@ -688,7 +688,7 @@ Scraper.prototype.scrape = function(fields){
 
         if( field.$el.find('.acf-taxonomy-field[data-type="multi_select"]').length > 0 ){
 
-            var select2Target = (helper.acf_version >= 5.6)?'select':'input';
+            var select2Target = (acf.select2.version >= 4)?'select':'input';
```

Comment:
Maybe, it is nicer to have a global `minimalRequiredSelect2Version` variable that contains the number `4`, that is easier to read than only using a magical `4` value.

Reply:
Hm...The things is that
1. This isn't a minimal required version, but rather a switch to work with any select2 version.
2. The number is compared to `acf.select2.version`, so isn't it pretty obvious that this is the select2 version?